### PR TITLE
Integrate systemd configuration in rpi-raspbian image

### PR DESCRIPTION
**Pull request by nghiant2710**

### CODE REVIEW

File: mkimage.sh [hunk 1]
```diff
@@ -103,6 +103,15 @@ LABEL io.resin.architecture="rpi" \
 	  io.resin.device-type="raspberrypi"
```
Comment (petrosagg) [inline mkimage.sh]: I propose we move the above Dockerfile in its own file

File: mkimage.sh [hunk 2]
```diff
@@ -103,6 +103,15 @@ LABEL io.resin.architecture="rpi" \
...
 EOF
 
+# Systemd
+if [ $SUITE == 'wheezy' ]; then
+	cat Dockerfile.no-systemd.partial >> "$dir/Dockerfile"
```
Comment (petrosagg) [inline mkimage.sh]: Can we move this partial in a `Dockerfile.tpl` and `Dockerfile.nosystemd.tpl`?

File: mkimage.sh (hunk 1 above)
Comment (nghiant2710) [inline mkimage.sh]: updated!

File: mkimage.sh (hunk 2 above)
Comment (nghiant2710) [inline mkimage.sh]: updated!

**Pull request merged**